Cpp/fostgres-fg/fg.testserver.cpp

```diff
@@ -50,8 +50,14 @@ namespace {
                     fostlib::coerce<fostlib::string>(*h));
         }
         try {
-            return fostlib::urlhandler::view::execute(
+            auto view_result = fostlib::urlhandler::view::execute(
                     fg::json("fg.test"), path.substr(1), req, host);
+            if (req.method() == "HEAD") {
+                // HEAD returns empty body so we will not use the view result response body
+                auto response = boost::make_shared<fostlib::empty_mime>();
```

Comment:
It would be best to copy the headers from the old response, but I guess if we're not looking at them yet then why bother?